repository-creator/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM python:3.7-slim-bullseye as base
```

Comment:
You could also use `openstax/python3-base:latest` which is python 3.8 bullseye.

Reply:
When I try `python 3.8`, I get some errors installing lxml and neb dependencies.

Reply:
ah ok. i think i vaguely remember something about that.. no worries then.